pkg/apis/core/validation/validation.go

```diff
@@ -923,7 +923,10 @@ var validVolumeDownwardAPIFieldPathExpressions = sets.NewString(
 	"metadata.namespace",
 	"metadata.labels",
 	"metadata.annotations",
-	"metadata.uid")
+	"metadata.uid",
+	"spec.nodeName",
+	"spec.serviceAccountName",
+	"status.hostIP")
```

Comment:
I expected to see status.podIP here.

Reply:
// TODO: support "status.podIP" once we can get pod ip before first setup volume.